From cb5dd4e7f70f8c68f9140cb275341dadd1c0ad46 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Fri, 23 Jun 2017 16:28:10 -0500
Subject: [PATCH 2017/5242] MLK-15284-4: dma: Kconfig: add MXS_DMA dependency
 for ARM64

commit  8f2b815378678d10f17aa34c4e57a567d97fc456 from
https://source.codeaurora.org/external/imx/linux-imx.git

add MXS_DMA dependency for ARCH_MXC_ARM64

The patch also merge the upstreamed change that extend the dependency to
allow the mxs dma driver to be built whenever ARCH_MXS or ARCH_MXC is
selected. Refer to
https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/drivers/dma/Kconfig?h=next-20170626&id=d762e4f35601239cbebfbfd43d99876d8f220927

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/dma/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/Kconfig b/drivers/dma/Kconfig
index 3fb98c6..20f8d51 100644
--- a/drivers/dma/Kconfig
+++ b/drivers/dma/Kconfig
@@ -393,7 +393,7 @@ config MV_XOR_V2
 
 config MXS_DMA
 	bool "MXS DMA support"
-	depends on ARCH_MXS || ARCH_MXC || COMPILE_TEST
+	depends on ARCH_MXS || ARCH_MXC || ARCH_MXC_ARM64 || COMPILE_TEST
 	select STMP_DEVICE
 	select DMA_ENGINE
 	help
-- 
1.7.9.5

